packages/node_modules/@webex/webex-core/src/lib/services/services.js

```diff
@@ -702,49 +775,20 @@ const Services = WebexPlugin.extend({
 
     // Listen for configuration changes once.
     this.listenToOnce(this.webex, 'change:config', () => {
-      // Destructure the configuration object for the services plugin.
-      const {services} = this.webex.config;
-
-      // Validate that the services configuration exists.
-      if (services) {
-        // Check for discovery services.
-        if (services.discovery) {
-          // Format the discovery configuration into an injectable array.
-          const formattedDiscoveryServices = Object.keys(services.discovery)
-            .map((key) => ({
-              name: key,
-              defaultUrl: services.discovery[key]
-            }));
-
-          // Inject formatted services into services catalog.
-          catalog.updateServiceUrls('discovery', formattedDiscoveryServices);
-        }
-
-        // Check for allowed host domains.
-        if (services.allowedDomains) {
-          // Store the allowed domains as a property of the catalog.
-          catalog.setAllowedDomains(services.allowedDomains);
-        }
-
-        // Set `validateDomains` property to match configuration
-        this.validateDomains = services.validateDomains;
-      }
+      this.initConfig();
     });
 
     // wait for webex instance to be ready before attempting
     // to update the service catalogs
     this.listenToOnce(this.webex, 'ready', () => {
-      /* eslint-disable camelcase */
-      if (this.webex.credentials.canAuthorize) {
-        this.updateServices()
-        // this catch prevents crashing in unique situations found
-        // primarily in unit testing with the karma suite.
-          .catch(
-            () => this.logger.warn('services: catalog retrieval failed w/auth')
-          );
-      }
-      else {
-        this.collectPreauthCatalog({email: this.webex.config.email});
```

Comment:
is this part still needed? was it added for the guest devices client or was it for something else?

Reply:
This was added a while ago as a feature that was never documented or utilized in the SDK. A developer would need to do the following to use this:
```js
const webex = new Webex({
  email: 'example@website.com'
});
```
Since this was never documented, let us know if you'd like us to re-implement this as a part of the PR.

Reply:
it was not utilized by the web client. @robstax can you confirm if this is needed for guest client?